packages/manager/apps/pci-file-storage/src/components/breadcrumb/__tests__/FileStorageBreadcrumb.component.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ComponentProps } from 'react';
+
+import { MemoryRouter } from 'react-router-dom';
+
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import { BreadcrumbItemProp } from '@ovhcloud/ods-react';
+
+import { FileStorageBreadcrumb } from '@/components/breadcrumb/FileStorageBreadcrumb.component';
+
+vi.mock('@ovhcloud/ods-react', () => {
```

Comment:
I'm not sure to understand why you mock ODS components here.
I think you should not as if ODS breaks  this component, your test will always be successful whereas UI could actually be broken.